common/types/shared.go

```diff
@@ -2142,6 +2142,20 @@ type ActiveClusterSelectionPolicy struct {
 	ExternalEntityKey  string `json:"externalEntityKey,omitempty"`
 }
 
+func (p *ActiveClusterSelectionPolicy) Equals(other *ActiveClusterSelectionPolicy) bool {
```

Comment:
remark: I find it confusing the go language hasn't made equality a first-class interface.